src/Configuration/Parser/PermissionsParser.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bolt\Configuration\Parser;
+
+use Bolt\Common\Arr;
+use Tightenco\Collect\Support\Collection;
+
+class PermissionsParser extends BaseParser
+{
+    public function __construct(string $projectDir, string $filename = 'permissions.yaml')
+    {
+        parent::__construct($projectDir, $filename);
+    }
+
+    /**
+     * Read and parse the permissions configuration file.
+     */
+    public function parse(): Collection
+    {
+        $defaultConfig = $this->getDefaultConfig();
+        $yamlConfig = $this->parseConfigYaml($this->getInitialFilename());
+        $permissionConfig = Arr::replaceRecursive($defaultConfig, $yamlConfig);
+
+        return new Collection($permissionConfig);
+    }
+
+    protected function getDefaultConfig(): array
+    {
+        // TODO PERMISSIONS add more defaults
```

Comment:
@simongroenewolt on this one, I also suggest we make use of @bobdenotter 's recent `yaml-migrations` extension.

It allows us to update `bolt/project` files according to a release of `bolt/core`. We can use this here to make sure each project that updates to 4.2 will get a default `permissions.yaml`. As long as the new permissions are not a breaking change (which I don't think they will be), that might be the simplest way to insert them into existing projects.